### DISCUSSION TITLE
材质的tilingOffset.z值大于1之后在iphone7p ios13.6版本表现异常

### DISCUSSION THREAD

**Discussion by jasonxs45** [galacean/discussion#104] [Engine] [answered]
如题，参照官方给的flappy bird案例，在设置背景无限向左偏移时：this._groundMaterial.tilingOffset.z += deltaTime * this._groundHorizontalV;  当this._groundMaterial.tilingOffset.z值大于1之后，材质的纹理表现异常。看起来视觉效果是材质的tilingOffset.x被设置成0了，纹理呈现横线拉丝状
<img width="372" alt="image" src="https://github.com/galacean/discussion/assets/19587683/607b79b8-fbfd-453f-a73c-efe3f7b50b28">

再补充一下如果初始化设置tilingOffset.set(2, 1, 0, 0)，也会出现类似情况。

**Comment by cptbtptpbcptdtptp** [comment]
感谢提供信息，我们明天去机房找一下 IOS13.6 的机器测试一下。

**Reply by jasonxs45** [reply]
忘记标明版本了 0.9.20

**Comment by zhuxudong** [comment, marked as answer]
WebGL1.0 环境下，NP2(非2次幂) 的纹理无法设置`wrapMode`为 `Repeat`，检查下是否是这个原因，如果是的话，可以将图片变成二次幂
